gradle.properties

```diff
@@ -1,4 +1,4 @@
-version=29.32.5
+version=29.32.6
```

Comment:
This PR is introducing a new behavior - support ByteStringArray as finder param, I think it should be a minor version bump.